src/accel/detail/LocalTransporter.hh

```diff
@@ -0,0 +1,67 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2022 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file accel/detail/LocalTransporter.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <G4Track.hh>
+
+#include "corecel/Types.hh"
+#include "celeritas/global/CoreParams.hh"
+#include "celeritas/global/Stepper.hh"
+
+#include "../SetupOptions.hh"
+
+namespace celeritas
+{
+namespace detail
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Transport a set of primaries to completion.
+ */
+class LocalTransporter
+{
+  public:
+    //!@{
+    //! \name Type aliases
+    using SPCOptions = std::shared_ptr<const SetupOptions>;
+    using SPCParams  = std::shared_ptr<const CoreParams>;
```

Comment:
I know I was the one to put these type aliases in, but it looks like in the rest of the code we've been using `SPConstFoo`. Do you mind changing these? (Also `accel/RunAction.hh` needs fixing too.)